bitnami/openldap/2.5/debian-12/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -633,9 +634,7 @@ ldap_initialize() {
             ldap_add_custom_schemas
         fi
         # additional configuration
-        if [[ ! "$LDAP_PASSWORD_HASH" == "{SSHA}" ]]; then
-            ldap_configure_password_hash
-        fi
+        ldap_configure_password_hash
```

Comment:
I guess it does not hurt to always add those configs no matter the algorithm, right?

Reply:
Making it explicit should not hurt.